src/test/java/org/folio/list/service/refresh/RefreshSuccessCallbackTest.java

```diff
@@ -28,6 +29,9 @@ class RefreshSuccessCallbackTest {
   @Mock
   private ListContentsRepository listContentsRepository;
 
+  @Mock
```

Comment:
Is this doing anything?

Reply:
Not directly, but it's providing the required dependency for the RefreshSuccessCallback that's being tested. I didn't bother adding any assertions for it getting used, though